src/coreclr/jit/llvm.cpp

```diff
@@ -1639,6 +1645,65 @@ unsigned Llvm::buildMemCpy(Value* baseAddress, unsigned startOffset, unsigned en
     return size;
 }
 
+void Llvm::buildThrowException(llvm::IRBuilder<>& builder, const char* helperClass, const char * helperMethodName, Value* shadowStack)
+{
+    CORINFO_METHOD_HANDLE methodHandle = _getCompilerHelpersMethodHandle(_thisPtr, helperClass, strlen(helperClass),
+                                                                         helperMethodName, strlen(helperMethodName));
```

Comment:
```suggestion
    CORINFO_METHOD_HANDLE methodHandle = _getCompilerHelpersMethodHandle(_thisPtr, helperClass, helperMethodName);
```

Reply:
Thanks, using `Marshal.PtrToStringAnsi((IntPtr)className)` to go to a dotnet string